### PR TITLE
Added message component (Issue #3)

### PR DESCRIPTION
**Pull request by emmajam**
TO DO:
- [x] Colors
- [x] Sizes

### CODE REVIEW

File: catalog/level.js [hunk 1]
```diff
@@ -94,6 +94,6 @@ ${<TableSpecimen rows={parsePropTypes(Level)} />}
 
 ## Reference
 
-[https://bulma.io/documentation/layout/hero/](https://bulma.io/documentation/layout/hero/)
+[https://bulma.io/documentation/layout/level/](https://bulma.io/documentation/layout/level/)
```
Comment (jxom) [inline catalog/level.js]: Is this meant to be message?

File: catalog/message.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
+import React from 'react';
```
Comment (jxom) [inline catalog/message.js]: nice docs

File: src/Message/Message.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        {children}
+      </div>
+    );
+  };
+}
+
+Message.propTypes = {
+  children: PropTypes.node.isRequired,
+  className: PropTypes.string,
+  color: PropTypes.string,
```
Comment (jxom) [inline src/Message/Message.js]: Can use `color` from https://github.com/jxom/rebul/blob/master/src/_prop-types.js

File: src/Message/Message.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  textSize: PropTypes.string
```
Comment (jxom) [inline src/Message/Message.js]: can use size from https://github.com/jxom/rebul/blob/master/src/_prop-types.js

File: catalog/level.js (hunk 1 above)
Comment (emmajam) [reply catalog/level.js]: this is in the `level.js` file, you got it wrong.

Comment (jxom) [reply catalog/level.js]: Imma fkn idiot

Comment (emmajam) [reply catalog/level.js]: hehe 🙊

**Pull request merged**